src/fundamental/macro-fundamental.h

```diff
@@ -557,6 +557,12 @@ static inline uint64_t ALIGN_OFFSET_U64(uint64_t l, uint64_t ali) {
                 }                                                       \
         }
 
+typedef struct {
+        int _empty[0];
```

Comment:
Is it possible to drop zero here?

Reply:
Maybe, but I'd prefer to keep this PR strictly about includes, it's already large enough as it is